src/client/Componets/login/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './index.scss';
+import LoginForm from '../loginForm/index.js';
+
+class Login extends React.Component {
+  render() {
+    return (
+      <main>
+        <header>
+          <img src="../../img/black_pearl.png" alt="Black_pearl"/>
+          <h1>Welcome Onboard!</h1>
+        </header>
+        <LoginForm />
+        <p>Don`t have an account?<a href=""> Sign up>></a></p>
```

Comment:
`<p>Don`t have an account? <a href="#">Sign up>></a></p>`

also remove the `>>` from the end